source/common/quic/quic_filter_manager_connection_impl.cc

```diff
@@ -97,6 +99,7 @@ void QuicFilterManagerConnectionImpl::close(Network::ConnectionCloseType type) {
     } else {
       delayed_close_state_ = DelayedCloseState::CloseAfterFlush;
     }
+    std::cerr << "========= danzh: delay close\n";
```

Comment:
Presumably you're still debugging the CI failures?

Reply:
Yes, this is the only way I could think of to debug windows.